src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/Extensions/OpenApiHostBuilderExtensions.cs

```diff
@@ -20,7 +21,8 @@ public static IHostBuilder ConfigureOpenApi(this IHostBuilder hostBuilder)
             hostBuilder.ConfigureServices(services =>
             {
                 services.AddSingleton<IOpenApiHttpTriggerContext, OpenApiHttpTriggerContext>();
-                services.AddSingleton<IOpenApiTriggerFunction, OpenApiTriggerFunction>();
+                services.AddSingleton<IOpenApiTriggerFunction, OpenApiTriggerFunction>();
+                services.AddSingleton<IOpenApiConfigurationOptions, DefaultOpenApiConfigurationOptions>();
```

Comment:
I don't think this line is necessary because the `IOpenApiConfigurationOptions` instance is injected from the other place &ndash; `Program.cs`. Otherwise, there will be two `IOpenApiConfigurationOptions` instances &ndash; one for `DefaultOpenApiConfigurationOptions` and the other for dynamically generated one.